### PR TITLE
Print logs

### PR DESCRIPTION
**Pull request by sgkim126**

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -134,6 +134,8 @@ fn main() -> Result<(), String> {
             service_fn(move |req| get_work(req, &config, submit_port))
         })
         .map_err(|e| error!("server error: {}", e));
+    info!("Server started. It listens on {:?}", addr);
```
Comment (kseo) [inline src/main.rs]: -> Server started, listening on

File: src/worker/blake.rs [hunk 2]
```diff
@@ -68,11 +68,14 @@ impl Worker for BlakeWorker {
 
         self.is_executed = true;
         let hash = blake256(&self.message);
-        if U256::from(hash) <= self.target {
+        let current_score = U256::from(hash);
+        if current_score <= self.target {
+            info!("Solution found");
```
Comment (kseo) [inline src/worker/blake.rs]: Why don't you print the found solution?

File: src/worker/work.rs [hunk 3]
```diff
@@ -45,11 +45,15 @@ pub fn work(id: usize, hash: &H256, target: &U256, mut worker: Box<Worker>) -> O
...
-                Some(solution) => return Some(solution),
+                Some(solution) => {
+                    info!("Nonce: {}", nonce);
+                    return Some(solution)
+                }
                 None => {}
             }
         }
     }
+    info!("Cannot find the solution for hash {}", hash);
```
Comment (kseo) [inline src/worker/work.rs]: Cannot -> Could not

File: src/worker/work.rs [hunk 4]
```diff
@@ -70,6 +74,7 @@ pub fn submit(hash: H256, solution: Vec<Vec<u8>>, port: u16) {
     *req.uri_mut() = format!("http://127.0.0.1:{}", port).parse().unwrap();
     req.headers_mut().insert("content-type", HeaderValue::from_str("application/json").unwrap());
 
+    info!("Job finish with hash {}, seal {:?}", hash, seal);
```
Comment (kseo) [inline src/worker/work.rs]: finish -> finished

**Review by kseo — CHANGES_REQUESTED** [review]

**Comment by sgkim126** [comment]
@kseo PR updated

**Review by kseo — APPROVED** [review]
Ok

**Pull request merged**